SU-SOE/stanford_soe_helper
# Redesign cjw

# READY FOR REVIEW

# Summary
This adds a bunch of stuff!

# Needed By (Date)
Soon. We want it on preview to demo

# Urgency
Yes

# Steps to Test

1. Pull the latest on  https://github.com/SU-SWS/stanford_magazine
2. Pull the latest dev branch of stanford_soe_helper (in this case: redesign-cjw)
3. Enable stanford_magazine_issue_views
4. Revert stanford_magazine
5. Delete the variable:
$ drush eval 'variable_del("stanford_soe_helper_magazine_settings")'
6. Verify:
- The _Recent Issues_ block displays on magazine_issue pages
- The _All Digital Magazine Issues_ block displays on all magazine_issue pages
- The functionality of the _All Digital Magazine Issues_ block (edit and save)
- Looks right at /magazine (https://projects.invisionapp.com/d/main#/console/10423578/230196822/preview)
-  Looks right at /magazine/all (https://projects.invisionapp.com/d/main#/console/10423578/230198361/preview)
-  Page 1-9 display from _Stanford Magazine Articles_ has been deleted
- New _Stanford Magazine Article: Most Recent_ view
- All the new styles

# Affected Projects or Products
- SoE

# Associated Issues and/or People
- _Previous Issues_: https://stanfordits.atlassian.net/browse/SOE-1859
- _DM Landing Page_: https://stanfordits.atlassian.net/browse/SOE-1619
- https://github.com/SU-SWS/stanford_magazine
## PRs
https://github.com/SU-SWS/stanford_magazine/pull/3
- Anyone who should be notified? @josephgknox @boznik 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
